lib/members.ex

```diff
@@ -126,4 +126,63 @@ defmodule GlassFactoryApi.Members do
       {:error, error} -> raise error
     end
   end
+
+  @doc """
+  Returns the active members from an organization.
+
+  ## Examples
+
+      iex> GlassFactoryApi.Members.list_active_members()
+      {:ok, [ %Member{
+            name: "John Doe",
+            email: "john.doe@example.org",
+            id: 0,
+            archived: false,
+            capacity: 8.0,
+            freelancer: false,
+            joined_at: "2019-01-01"
+          }
+        ]
+      }
+
+      iex> GlassFactoryApi.Members.list_members()
+      {:error, "econnrefused"}
+  """
+  @spec list_active_members(map()) :: {atom, list(Member)}
```

Comment:
We should run dialyxir in the project, I'm pretty sure this spec will fail in the error case, because the error won't be a list of members. We can fix that with:

```suggestion
  @spec list_active_members(map()) :: {:ok, list(Member)} | {:error, any}
```

BTW, I think it's a good idea to define a type to better document what kind of options we accept in the `config` map.

Reply:
I'm updating this now and will setup dialyxir and fix all occurrences in a next PR, WDYT?

Reply:
Great! :shipit: